ckanext/vocabulary_services/cli.py

```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
 
+import ckan.plugins.toolkit as toolkit
 import click
 import logging
 
+from ckan.common import config
```

Comment:
Hi @awset,
You can you the toolkit.config to use the config, you do not need to import from ckan.common and it's encouraged to use the toolkit as much as possible in extensions instead of directly importing ckan files.

Reply:
@awset I refactored the code.